### ISSUE TITLE
Fix maven deploy verification step in release workflow

### ISSUE THREAD

**Issue by EricWittmann** [Apicurio/apicurio-registry#861]
The 1.3.1.Final release worked (including deployment to maven central) but the verification step failed.  I believe because there is a delay between the completion of deployment and when artifacts are actually available in maven central.  We may need to skip verification, delay before verifying, or find another way.

**Comment by riprasad** [comment]
@EricWittmann Looks like it takes a while for the artifact to actually become available in maven central. It works when I tried in local now

**Comment by EricWittmann** [comment]
Yeah I think that's exactly right.  I'll probably comment out that verification step for now until we can figure something out.
